osf/models/metaschema.py

```diff
@@ -12,19 +11,19 @@
 from website.project.metadata.utils import create_jsonschema_from_metaschema
 
 FORMBLOCK_TYPES = [
-    ('string', 'string'),
-    ('singleselect', 'singleselect'),
-    ('multiselect', 'multiselect'),
-    ('osf-author-import', 'osf-author-import'),
-    ('osf-upload', 'osf-upload'),
-    ('header', 'header'),
-]
-
-FORMBLOCK_SIZES = [
-    ('sm', 'sm'),
-    ('md', 'md'),
-    ('lg', 'lg'),
-    ('xl', 'xl'),
+    ('page-heading', 'page-heading'),
+    ('section-heading', 'section-heading'),
+    ('subsection-heading', 'subsection-heading'),
+    ('paragraph', 'paragraph'),
+    ('question-title', 'question-title'),
+    ('short-text-input', 'short-text-input'),
+    ('long-text-input', 'long-text-input'),
+    ('file-input', 'file-input'),
+    ('contributors-input', 'contributors-input'),
+    ('single-select-input', 'single-select-input'),
+    ('multi-select-input', 'multi-select-input'),
+    ('select-input-option', 'select-input-option'),
+    ('select-other-option', 'select-other-option'),
```

Comment:
Am not using this `block_type` yet, `select-other-option`.  Every existing schema, if there's an "Other" option, where additional information is required, is an entirely different question altogether.  No current "Other - Please explain" answers are stored as part of the original question, but are in a follow-up question.   So currently, the "other" choice, is a `select-input-option` like all the other choices, and a followup-other question is a `long-text-input`.

Reply:
Example of what I'm talking about, where two schema questions are related:
```json
{
            "qid": "study_type",
            "title": "Finally. For record keeping purposes, please tell us the type of study you are pre-registering.",
            "nav": "Study type",
            "type": "choose",
            "format": "singleselect",
            "options": [
                "Class project or assignment",
                "Experiment",
                "Survey",
                "Observational/archival study",
                "Other (describe below)"
            ]
        },
        {
            "qid": "study_type_other",
            "title": "Other",
            "nav": "Other",
            "type": "string",
            "format": "textarea"
        }
```

Reply:
`select-other-option` maybe deferred improvement for now